content/windows/docs/ami.md

```diff
@@ -27,6 +27,13 @@ Use Amazon EC2 Image Builder to select between Windows Server versions, AWS Wind
 
 **NOTE:** Prior to selecting a base image, consult the [Windows Server Version and License](licensing.md) section for important details pertaining to release channel updates.
 
+## Configuring faster launching for custom EKS optimized AMIs ##
+
+When using a custom EKS Optimized AMI, Windows worker nodes can be launched up to 65% faster by enabling the Fast Launch feature. This feature maintains a set of pre-provisioned snapshots which have the _Sysprep specialize_, _Windows Out of Box Experience (OOBE)_ steps and required reboots already completed. These snapshots are then used on subsequent launches, reducing the time to scale-out or replace nodes. Fast Launch can be only enabled for AMIs *you own* through the EC2 console or in the AWS CLI and the number of snapshots maintained is configurable. 
+> :warning: **Fast Launch is not compatible with the default Amazon-provided EKS optimized AMI**, create a custom AMI as above before attempting to enable it. 
```

Comment:
To write an admonition you use `!!! warning` You can see some examples [here](https://python-markdown.github.io/extensions/admonition/)